es-ds-components/components/es-dropdown.vue

```diff
@@ -0,0 +1,82 @@
+<script setup lang="ts">
+import Dropdown from 'primevue/dropdown';
+
+interface Props {
+    disabled?: boolean;
+    modelValue?: any;
+    options?: string[];
+    placeholder?: string;
+    title?: string;
+}
+const props = withDefaults(defineProps<Props>(), {
+    disabled: false,
+    modelValue: undefined,
+    options: () => [],
+    placeholder: '',
+});
+
+const emit = defineEmits(['update:modelValue']);
+
+const isOpen = ref(false);
+
+const isSelected = (option: any) => option === props.modelValue;
+
+const id = useId();
+</script>
+
+<template>
+    <div class="dropdown-wrapper">
+        <label
+            v-if="props.title"
+            :for="id">
+            {{ props.title }}
+        </label>
+        <label
+            v-else
+            :for="id"
+            class="sr-only">
+            Select an option
+        </label>
+        <dropdown
+            :id="id"
```

Comment:
I noticed that clicking the label doesn't automatically give focus to the field, which it should. Inspecting the DOM shows that this id is applied to the `<div>` wrapper rather than the `<span tabindex="0">` that is focusable. Per the [PrimeVue Dropdown API Docs](https://v3.primevue.org/dropdown/), it looks like this should be `inputId` instead of `id` - that might solve it?